src/pages/Regions/Region.tsx

```diff
@@ -85,6 +85,7 @@ const Region = () => {
 
   const [visibleDrawer, setVisibleDrawer] = useState(false);
   const [admins, setAdmins] = useState<any[]>([]);
+  const [sixAdmins, setSixAdmins] = useState<any[]>([]);
```

Comment:
if we would need 5 in the future?? Maybe change the name?

Reply:
I don't think so, because this variable is using at place where we need exactly six elements. Before this change, only six admins were written at the variable "admins", was it correct??

Reply:
Well, ok, that's normal than